presentation/src/main/java/com/gulderbone/presentation/nutritionmain/NutritionMainFragment.kt

```diff
@@ -0,0 +1,14 @@
+package com.gulderbone.presentation.nutritionmain
+
+import androidx.fragment.app.Fragment
+import com.gulderbone.presentation.R
+import com.gulderbone.presentation.databinding.FragmentNutritionMainBinding
+import com.gulderbone.presentation.viewBinding
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class NutritionMainFragment : Fragment(R.layout.fragment_nutrition_main) {
```

Comment:
```suggestion
internal class MainFragment : Fragment(R.layout.fragment_main) {
```
Or more expressive name

Reply:
Changed, thanks!